index.html

```diff
@@ -0,0 +1,193 @@
+                    <ul class="nav-list">
+                        <li><a href="./about.html"> About</a></li>
+                        <li><a id="coursesiconindex" href="#main-courses-section"> Courses</a></li>
+                        <li><a id="joiniconindex" href="#"> Join</a></li>
+                        <li><a id="sponsericonindex" href="#"> Sponser</a></li>
+                        <li><a id="latesticonindex" href="#"> Latest</a></li>
+                        <li><a href="#"><button class="nav-bar-btn">Register Now!</button></a></li>
+                    </ul>
+                </div>     
+             
+             </nav>
+        </div>
+     </div>
+</heder>
+     <!-- Main home page section start from here -->
+    <section class="contianer-fluid home-page-fluid-container">
+        <div class="container-fluid menu-bar-fluid">
+            <div class="row">
+                <div id="wrap" class="wrap">
+                    <div id="bar1" class="menu"></div>
+                    <div id="bar2" class="menu"></div>
+                    <div id="bar3" class="menu"></div>
+                </div>
+            </div>
+            <div id="navlistdisplay1" class="row">
+                <ul class="nav-list-mobile">
+                    <li><a href="./about.html"> About</a></li>
+                    <li><a id="coursesiconmobile" href="#main-courses-section">Main Countries</a></li>
+                    <li><a id="joiniconmobile" href="#"> Join</a></li>
+                    <li><a id="sponsericonmobile" href="#"> Sponser</a></li>
+                    <li><a id="latesticonmobile" href="#"> Latest</a></li>
+                </ul>
+            </div>
+        </div>
+
+      
+
+        <div class="container main-home-container">
+            <div class="row heading-1">
+                <h1>"Welcome!"</h1>
+            </div>
+            <div class="row heading-2">
+                <h2>AFRICAN DANCE CONFERENCE<br> 2023</h2>
+            </div>
+            <div class="row para-heading">
+                <p>A joyful celebration is going to start soon By the opening of classes. Thereby, creating a change in education environment by giving admission to students from 180 countries.</p>
+            </div>
+            <div class="row heading-3">
+                <h3>2023.10.15(THU) ~ (FRI)</h3>
+            </div>
+            <div class="row heading-4">
+                <h4>@Africa Dance Foundation, And many more</h4>
+            </div>
+
+        </div>
+    </section>
+
+    <!-- Main Courses section start from here -->
+    <section id="main-courses-section" class="contianer-fluid main-program">
+        <div class="container main-program-container ">
+            <div class="row">
+                <div class="col-12 d-flex justify-content-center main-text">Contributing Countries</div>
+            </div>
+            <div class="row d-flex justify-content-center">
+                <div class="col-1 main-text-line">line</div>
+            </div>
```

Comment:
- [ ] Please make sure the `Contributing Countries` section doesn't overlap the headline section. 

**Your project design**
![image](https://user-images.githubusercontent.com/74411135/235281048-3ae036ed-f17d-48c5-9eab-b05dc2550b72.png)

**your project when resized**
![image](https://user-images.githubusercontent.com/74411135/235281134-0940e79c-246b-411c-807e-df1887a13dbf.png)

This will make your project look way prettier 😄 🌹

Reply:
@Trast00 , I am struggling to get this overlapping effect I don't seem to reproduce it on my  mobile sizes. Any clue of the screen size I should look at?

Cheers

David